django_esidoc/middleware.py

```diff
@@ -4,9 +4,11 @@
 from django.conf import settings
 from django.contrib.auth import login, authenticate
 
+from xml.etree import ElementTree
+from xml.etree.ElementTree import ParseError
+
 from .utils import (
```

Comment:
Can be on one line

Reply:
Fixed by 46dab54

Reply:
Uhm, does not seem so.

Reply:
But it doesn't really matter as it will be blacked!

Reply:
Fixed by c808d83